experiments/AtmosLES/dycoms.jl

```diff
@@ -242,7 +242,7 @@ function init_dycoms!(bl, state, aux, (x,y,z), t)
     z = altitude(bl.orientation, aux)
 
     # These constants are those used by Stevens et al. (2005)
```

Comment:
This value 8.5e-3 is not the same as that defined in the paper ? Would suggest removing this comment in that case.